### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by lucyleeow**
PR mostly just consists typos. Ping @lesteve, in case you miss it.

### CODE REVIEW

File: python_scripts/04_basic_parameters_tuning.py [hunk 1]
```diff
@@ -278,13 +278,13 @@
 # %% [markdown]
 # ## Combining evaluation and hyper-parameters search
 #
-# Cross-validation was used for searching the best model parameters. We
-# previously evaluate model performance through cross-validation as well. If we
+# Cross-validation was used for searching for the best model parameters. We
+# previously we evaluated model performance through cross-validation as well. If we
```
Comment (lesteve) [inline python_scripts/04_basic_parameters_tuning.py]: There is a we already on the previous line!

Comment (lesteve) [reply python_scripts/04_basic_parameters_tuning.py]: I will remove it, no worries!

Comment (lucyleeow) [reply python_scripts/04_basic_parameters_tuning.py]: whoops....

**Comment by lesteve** [comment]
Very nice! Thanks a lot for reviving the PR you did on the EuroScipy tutorial repo !

You need to update the notebooks as well. It is not super convenient right now but in an ideal world doing `make` should just work if you have the needed dependencies in your base conda environment. Personally I have a specific conda environment (called `scikit-learn-tutorial`) with the needed dependency (e.g. Jupytext) so I do:
```
make -e JUPYTER_KERNEL=scikit-learn-tutorial 
```

I would be enclined to merge your typos/grammar/wording fixes first and discuss the more pedagogical issues on a separate issue.

**Comment by lucyleeow** [comment]
Oh that is a good point, I will make an issue for the other issues.

@glemaitre mentioned that there is a script that generates the notebooks? How do I use this....? :)

**Comment by lesteve** [comment]
Sorry I type Control-Enter to quickly, I have edited my previous message with the instructions.

**Comment by lucyleeow** [comment]
Not sure if I did it correctly, I got [this](https://github.com/voila-dashboards/voila/issues/267) error running `make -e JUPYTER_KERNEL=scikit-learn-tutorial` with the conda env created using the `environment.yml` file.

`jupyter kernelspec list` gave:
```
  python3    /home/lucy/miniconda3/envs/scikit-learn-tutorial/share/jupyter/kernels/python3
```

so I used `make -e JUPYTER_KERNEL=python3` instead and I think it worked.

Note the ``.py`` files now specify a newer version of `jupytext_version`

**Comment by lesteve** [comment]
Forgot about this one, thanks a lot for the fixes!

**Pull request merged**